src/index.js

```diff
@@ -637,6 +697,21 @@ function wrapIndyCallWithErrorHandling(func) {
         throw e
       }
 
+      if (Platform.OS === 'ios') {
+        const { indyCode, message } = parse
+        if (!isNaN(indyCode) && indyErrors.hasOwnProperty(indyCode)) {
+          const indyName = indyErrors[indyCode]
+          const indyErrorFromIOS = {
+            name: 'IndyError',
+            message,
+            indyCode: indyCode,
+            indyName: indyName,
+            indyCurrentErrorJson: null,
+          }
```

Comment:
I made the same mistake for the Android wrapper but this is throwing an object, not an error. Indy for node throws an `IndyError` error class instance. I think we could just copy the `IndyError` from indy-sdk (https://github.com/hyperledger/indy-sdk/blob/master/wrappers/nodejs/src/IndyError.js) and maybe update it a bit to use the normal class syntax (semantically the same: https://github.com/hyperledger/indy-sdk/blob/791b12162607b5ee2e7a87648c5ba486b8fbfdd1/wrappers/nodejs/src/IndyError.js)

Reply:
But I'm fine with merging this now (it is an edge case of handling when you use `instanceof`) and adding the class in a new PR